rudder-server-relay/SOURCES/relay-api/relay_api/shared_files.py

```diff
@@ -43,11 +43,11 @@ def parse_ttl(string):
     days = 0
     seconds = int(m.group(1))
   else:
-    daymatch  = r'(\d+)d(?:ays?)?'
-    hourmatch = r'(\d+)h(?:ours?)?'
-    minmatch  = r'(\d+)m(?:inutes?)?'
-    secmatch  = r'(\d+)s(?:econds?)?'
-    match = r'^\s*(?:' + daymatch + r'\s*)?(?:' + hourmatch + r'\s*)?(?:' + minmatch + r'\s*)?(?:' + secmatch + r'\s*)?$'
+    daymatch  = r'(\d+)\s*d(?:ays?)?'
```

Comment:
I had to remove the \s* from the () as it was captured and failed at int(m.group)